adding rag exercises to lab-3 (#20)
IBM/watsonx-prompt-lab: docs/lab-2/README.md

### 2.1 How to add grounding data:

> First, obtain the document you want to add.  For example, download to your laptop the following github issue example:

Reviewer: ```suggestion
First, obtain the document you want to add.  You can download the example GitHub issue provided:
```

Author: sounds good, thanks Rafael
